### DISCUSSION TITLE
How to get blocked ips?

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [kickstarter/rack-attack#584] [Questions (Q&A)]
Is there any way to get a list of all blocked ips?

**Comment by matiasmasca** [comment]
if you use Redis, you could do something like this:
` ips = Rack::Attack.cache.store.keys.find_all { |x| x.include?('rack::attack') }.sort`

**Comment by h0jeZvgoxFepBQ2C** [comment]
Ah nice, and is there an integrated way to delete an ip?
